packages/core/README.md

```diff
@@ -176,6 +176,12 @@ export type Notify = {
     event: EthereumTransactionData
   ) => TransactionHandlerReturn
   position: CommonPositions
+  replacement?: {
+    gasPriceProbability: {
+      speedup: number // default 80
```

Comment:
Might be helpful to add info on what the `speedup` and `cancel` props do in the configuration

Reply:
Yeah good call, added some more info.